.buildconfig.yml

```diff
@@ -32,6 +32,10 @@ projects:
     path: components/concept/sync
     description: 'An abstract definition of a browser data synchronization layer.'
     publish: true
+  concept-crash:
+    path: components/concept/crash
+    description: 'An abstract definition of a crash reporting component.'
+    publish: true
```

Comment:
Your commit message mentions `support-base`, but you created a separate concept component here. I think moving this into `support-base` is a good idea.

Reply:
Sure, I'll move it.